### DISCUSSION TITLE
Base Email template for same appearance

### DISCUSSION THREAD

**Discussion by razi-ahmad** [keycloak/keycloak#8687] [Ideas]
Having one basic email theme and all the rest extending it will give more flexibility and reduce code duplication

**Comment by stianst** [comment]
Can you provide some more details please?

**Reply by razi-ahmad** [reply]
Currently, If we have to add header and footer in the template then we have to update all email templates. So, for this solution, there can be one base template like login have. So, in this case no need to change all templates instead of changing only base template according to requirements( mostly, in the footer to show logos, alignment of text in the centre or something else )

**Reply by stianst** [reply]
Makes sense
